architecture/kubernetes_infrastructure.adoc

```diff
@@ -9,43 +9,41 @@
 
 toc::[]
 
-Kubernetes is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenance, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications. 
+== Overview
+http://kubernetes.io[Kubernetes] is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenance, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications.
```

Comment:
[Suggestion]
Kubernetes is used to manage containerized applications across a set of containers or hosts and provide deployment, maintenance, and application-scaling mechanisms.